### ISSUE TITLE
Duplicate of organizeByTags function on p 171-172

### ISSUE THREAD

**Issue by semmypurewal** [semmypurewal/LearningWebAppDev#46]
reported by Ken Hommel:

Why are there two definitions for the function "organizeByTags", one inside of main() and one before it? They have different function signatures; does JavaScript support polymorphism? 

If the first version of this function is #1 and the second version is #2, then #1 has a formal parameter and #2 does not. I don't know JavaScript; I would expect that the call to "organizeByTags(toDoObjects)" (in main()) would cause the interpreter to call #1, because the function signatures match. It doesn't, it calls #2. I'm guessing that this is a result of scoping, and polymorphism either doesn't exist in JavaScript or doesn't work the way I expect. 

In short, I think the first definition of organizeByTags is just an error, and should be omitted.
